apps/server/src/shared/domain/entity/dashboardElement.entity.spec.ts

```diff
@@ -126,17 +127,18 @@ describe('dashboardElement', () => {
 
 	describe('addReferences', () => {
 		describe('when Element has a single reference', () => {
-			it('should append references', () => {
+			it('should append references and set default group title', () => {
```

Comment:
you can think about seperating this into two tests (same setup, but different expects)